Packages/com.unity.inputsystem/InputSystem/API/Input.cs

```diff
@@ -804,7 +847,12 @@ private static Vector2 NormalizeAxis(Vector2 axis, float deadzone)
         public static Vector2 GetAxis(Inputs left, Inputs right, Inputs up, Inputs down)
         {
             var axis = GetAxisRaw(left, right, up, down);
-            var deadzone = GetInputSystemDefaultDeadZone(); // TODO account for controls being on gamepads
+            var deadzone = (GetDeviceTypeForInput(left) == InputDeviceType.Gamepad
```

Comment:
Does this makes sense? It seems like its intended to get dead zone from gamepad slot, but Inputs given above might come from e.g. 4 different gamepads? what dead zone should be used? Feels like deadzone should be passed to this function to move that decision higher up or have some restriction in place?

Reply:
Fixed by removing gamepad deadzone here